cmd/v/help/repl.txt

```diff
@@ -0,0 +1,3 @@
+Usage: v repl
```

Comment:
Interesting... I didn't know there was an actual, separate command.  Just running "v" with no options also starts the REPL.

Reply:
it is an effect of the repl being implemented as a tool; just `v` is a shortcut for it, so that it behaves like just `python` or `js` etc

Reply:
And it is not builtin in V itself, because it requires readline, and is currently ~9KB by itself.

Reply:
I am not sure if we should document both ways to start it, but I do not see harm either.